src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerEndpoint.java

```diff
@@ -166,15 +179,51 @@ public boolean equals(Object obj) {
         }
         return true;
     }
-    
+
+
+    @Override
+    public DescriptorImpl getDescriptor() {
+        return (DescriptorImpl) super.getDescriptor();
+    }
+
     @Extension
     public static class DescriptorImpl extends Descriptor<DockerServerEndpoint> {
+
+        private String defaultHost;
+
+        public String getDefaultHost() {
+            return defaultHost;
+        }
+
+        public DescriptorImpl() {
+            load();
+        }
+
+        @Override
+        public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
+            defaultHost = json.getString("defaultHost");
+            save();
+            return true;
+        }
+
+        public String getDefaultDockerHost() {
```

Comment:
currently user don't need to do anything, as long as his ENV has DOCKER_HOST set. 
UX to force user to enter docker host URL to all jobs would be just bad